lib/BudgetCategories/budgetCategoryPrice.dart

```diff
@@ -1,5 +1,30 @@
-class BudgetCategoryPrice {
-  String? budgetName;
-  int? currentCategoryPrice;
-  int? totalCategoryPrice;
+import 'package:flutter/material.dart';
+
+
+String category = 'Food';
+String progressAmt = '249';
```

Comment:
Are we going to start using test variables before or after sprint 2? wondering if we're supposed to begin making user inputs for this stuff

Reply:
Jay is working on the forms for that, I want to get started on the variables after Jay does his (hopefully before the end of sprint 2) so we can actually test and see if it works. But I'm slowly making the skeleton for the methods and getters & setters